include/swift/Sema/Constraint.h

```diff
@@ -891,7 +891,8 @@ class Constraint final : public llvm::ilist_node<Constraint>,
   /// Print constraint placed on type and constraint properties.
   ///
   /// \c skipLocator skips printing of locators.
-  void print(llvm::raw_ostream &Out, SourceManager *sm, unsigned indent = 0,
+  void print(llvm::raw_ostream &Out, SourceManager *sm,
+             unsigned firstLineIndent = 0, unsigned indent = 0,
```

Comment:
If I remember correctly the idea here is that the `Out` is indented when it's passed in that's why `indent` is used only if the constraint spans multiple lines to be able to indent everything after `\n` so we need to make sure that this happens correctly instead of adding `firstLineIndent` here.